otter/test/utils.py

```diff
@@ -599,5 +601,19 @@ def resolve_retry_stubs(eff):
     separately to determine that the policy is as expected.
     """
     assert type(eff.intent) is Retry
-    is_error, intermediate_result = guard(resolve_stubs, eff.intent.effect)
+    try:
```

Comment:
`guard` became a private function implementation detail of effect. I knew it was a bit dodgy to use it here when I first added this code...